jdaviz/helpers.py

```diff
@@ -0,0 +1,28 @@
+import pathlib
+from specutils import Spectrum1D
+
+from .app import Application
+
+class ConfigHelper():
+    """The Base Helper Class
+    Provides shared abstracted helper methods to the user
+    """
+    def __init__(self, configuration=None):
+        self._app = Application(configuration=configuration)
+
+
+class SpecViz(ConfigHelper):
+    def __init__(self):
+        super().__init__(configuration="specviz")
+
+    def load_data(self, data, data_label, format=None):
```

Comment:
```suggestion
    def load_data(self, data, data_label='specviz-data', format=None):
```
The most basic use case shouldn't require the user to give it a name.  Note that we *might* want to change this later to something "smarter" that figures out a new name based on what's loaded, but this suggestion is probably sufficient as a starting point.